packages/sdk/src/controllers/ConnectorController.ts

```diff
@@ -66,9 +68,14 @@ export class ConnectorController {
      * @param registration registration object containing all details about the connector
      * @returns the Id of the newly created connector, this Id should be used going forward.
      */
-    register = async (registration: ConnectorRegistration) => {
+    register = async (registration: ConnectorRegistration | Next.ConnectorGrafxRegistration) => {

```

Comment:
Stable version of controller shouldn't import anything from `next` - instead you have to create a `next` version of controller and use a new type of params (Similar to variable controller)